bundles/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/handler/KM200ThingHandler.java

```diff
@@ -324,7 +335,7 @@ private void addChannels(KM200ServiceObject serObj, Thing thing, List<Channel> s
             switch (subKeyType) {
                 case "stringValue":
                     /* Creating an new channel type with capabilities from service */
-                    List<@NonNull StateOption> options = null;
```

Comment:
This class doesn't have a `@NonNullByDefault` so are you sure that removing `@NonNull` here is safe?

Reply:
Yes, I am. But it should not be to hard to add it on class level if you prefer.

Reply:
Might as well